tests/safety/test_defaults.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python3
```

Comment:
"defaults" is not an accurate name for these safety modes

Reply:
Copied the convention we use for all other safety modes/tests: `safety_defaults.h`: `test_defaults.py` so it seems clear to me. It also tests the default rx and fwd hook.